src/ansible_navigator/utils.py

```diff
@@ -342,12 +342,14 @@ def debug_log(directory: str, found: bool, description: str):
     exit_messages.append(ExitMessage(message=exit_msg))
     return messages, exit_messages, None
 
+
 def divmod_int(numerator: Union[int, float], denominator: Union[int, float]) -> Tuple[int, int]:
     """Return the result of divmod, as a tuple of integers."""
     quotient, remainder = divmod(numerator, denominator)
     return int(quotient), int(remainder)
 
-def human_time(seconds: int) -> str:
+
+def human_time(seconds: Union[int, float]) -> str:
```

Comment:
Submit this separately. It has nothing to do with Zuul.

Reply:
Looks like the correct fix is https://github.com/ansible/ansible-navigator/pull/641